src/main/java/com/notably/logic/parser/NotablyParser.java

```diff
@@ -12,15 +12,24 @@
 import com.notably.logic.commands.HelpCommand;
 import com.notably.logic.commands.NewCommand;
 import com.notably.logic.commands.OpenCommand;
+import com.notably.logic.correction.StringCorrectionEngine;
 import com.notably.logic.parser.exceptions.ParseException;
+import com.notably.model.Model;
 
 /**
  * Parse in users input and generate the respective Commands
  */
 public class NotablyParser {
 
     private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+    private static final List<String> COMMAND_LIST = List.of("new", "edit", "delete", "exit", "open", "help");
+    private Model notablyModel;
+    private StringCorrectionEngine correctionEngine;
```

Comment:
Let's add a newline above this instance fields group to increase readability? We can then logically separate the instance fields from the class fields.